biggraphite/drivers/hybrid.py

```diff
@@ -108,10 +112,10 @@ def insert_downsampled_points_async(self, metric, datapoints, on_done=None):
         # Updating metadata here would be too expensive
         self._data_accessor.insert_downsampled_points_async(metric, datapoints, on_done)
 
-    def map(self, callback, start_key=None, end_key=None, shard=0, nshards=1, errback=None):
```

Comment:
You probably want to keep "map" since it will get implemented at some point. Just add stats()